### DISCUSSION TITLE
Change repo type to Template

### DISCUSSION THREAD

**Discussion by filiptronicek** [upptime/upptime#11] [Ideas] [answered]
In the README, there is a mention of generating your own from this template, but this repo is unable to be generated from since it isn't a Template repository.
![image](https://user-images.githubusercontent.com/29888641/89729727-c1a41080-da38-11ea-9fa1-d124421b412e.png)

**Comment by AnandChowdhary** [comment, marked as answer]
Thanks for opening this issue, @filiptronicek! This project is a WIP and will be production-ready by tomorrow. I'll make sure I mark it as a template then!

**Comment by filiptronicek** [comment]
Awesome, excited for that!
